src/Console/InstallAppCommand.cs

```diff
@@ -23,6 +23,11 @@ public InstallAppCommand(string name, string description = null) : base(name, de
                 IsRequired = true
             });
 
+            Add(new Option<string>(new[] { "--app-version" }, Resources.APP_VERSION_HELP)
```

Comment:
Would you consider adding a shortcut like -av (app version)?

Reply:
We don't really have any kind of standard for doing these (besides maintaining the community standards like "-v" for "--verbose"), and I can't remember "-av" being something standard, so, yes, I can add it!